identity-service/src/utils/contentNodeIPCheck.js

```diff
@@ -0,0 +1,76 @@
+const { logger } = require('../logging')
+const { recoverWallet, signatureHasExpired } = require('./apiSigning')
+const audiusLibsWrapper = require('../audiusLibsInstance')
+
+const FIND_CONTENT_NODES_INTERVAL_MS = 10 * 60 * 1000
+
+const KNOWN_CONTENT_NODE_IP_ADDRESSES = new Set([])
```

Comment:
is this used?

Reply:
good catch. i upper-cased it and didn't change the call-site
knownContentNodeIPAddresses
will fix

Reply:
oh no, jk. it is used. passed by ref at the bottom

Reply:
Right but is a value set? I see it being passed in to the private function but the exported function only takes in two args and this isn't one of them